db/migrate/20190725140946_add_park_photo_to_parks.rb

```diff
@@ -0,0 +1,5 @@
+class AddParkPhotoToParks < ActiveRecord::Migration[5.2]
+  def change
+    add_column :parks, :park_photo, :string
```

Comment:
Add public/uploads to your git ignore and delete the two jpgs that are part of this commit